test/upgrade/postupgrade.go

```diff
@@ -140,32 +148,32 @@ func updateService(serviceName string, t *testing.T) {
 	routeURL := svc.Status.URL.URL()
 
 	t.Log("Check that we can hit the old service and get the old response.")
-	assertServiceResourcesUpdated(t, clients, names, routeURL, test.PizzaPlanetText1)
+	assertServiceResourcesUpdated(t, clients, names, routeURL, textBeforeUpdate)
 
 	t.Log("Updating the Service to use a different image")
-	newImage := ptest.ImagePath(test.PizzaPlanet2)
+	newImage := ptest.ImagePath(toImage)
 	if _, err := v1test.PatchService(t, clients, svc, rtesting.WithServiceImage(newImage)); err != nil {
 		t.Fatalf("Patch update for Service %s with new image %s failed: %v", names.Service, newImage, err)
 	}
 
 	t.Log("Since the Service was updated a new Revision will be created and the Service will be updated")
 	revisionName, err := v1test.WaitForServiceLatestRevision(clients, names)
 	if err != nil {
-		t.Fatalf("Service %s was not updated with the Revision for image %s: %v", names.Service, test.PizzaPlanet2, err)
+		t.Fatalf("Service %s was not updated with the Revision for image %s: %v", names.Service, toImage, err)
 	}
 	names.Revision = revisionName
 
 	t.Log("When the Service reports as Ready, everything should be ready.")
 	if err := v1test.WaitForServiceState(clients.ServingClient, names.Service, v1test.IsServiceReady, "ServiceIsReady"); err != nil {
 		t.Fatalf("The Service %s was not marked as Ready to serve traffic to Revision %s: %v", names.Service, names.Revision, err)
 	}
-	assertServiceResourcesUpdated(t, clients, names, routeURL, test.PizzaPlanetText2)
+	assertServiceResourcesUpdated(t, clients, names, routeURL, textAfterUpdate)
 }
 
 // CreateNewServicePostUpgradeTest verifies creating a new service after upgrade.
 func CreateNewServicePostUpgradeTest() pkgupgrade.Operation {
 	return pkgupgrade.NewOperation("CreateNewServicePostUpgradeTest", func(c pkgupgrade.Context) {
-		createNewService(postUpgradeServiceName, c.T)
+		createNewService("pizzaplanet-post-upgrade-service", c.T)
```

Comment:
same question, why not use the const?

Reply:
Because it is used only in a single place.

Reply:
I think it is fine to remove the constant. Since it is used in one place and its value suffix documents its usage there is no reason to make it a global constant (although not bad for Golang either). The name seems more of a detail to me. What matters in this context is the operation `CreateNewServicePostUpgradeTest`, unless I miss something here.